src/main/java/com/wootecam/festivals/domain/organization/controller/OrganizationController.java

```diff
@@ -0,0 +1,31 @@
+package com.wootecam.festivals.domain.organization.controller;
+
+import com.wootecam.festivals.domain.organization.dto.OrganizationCreateDto;
+import com.wootecam.festivals.domain.organization.dto.OrganizationIdDto;
+import com.wootecam.festivals.domain.organization.service.OrganizationService;
+import com.wootecam.festivals.global.api.ApiResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/organizations")
+@RequiredArgsConstructor
+public class OrganizationController {
+
+    private final OrganizationService organizationService;
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public ApiResponse<OrganizationIdDto> createOrganization(
+            @Valid @RequestBody OrganizationCreateDto organizationCreateDto) {
+        Long organizationId = organizationService.createOrganization(organizationCreateDto);
+
+        return ApiResponse.of(new OrganizationIdDto(organizationId));
+    }
```

Comment:
p4
```suggestion
    @PostMapping
    public ApiResponse<OrganizationIdDto> createOrganization(
            @Valid @RequestBody OrganizationCreateDto organizationCreateDto) {
        return ResponseEntity.
          status(HttpStatus.CREATED)
          body(ApiResponse.of(new OrganizationIdDto(organizationId)));
    }
```
지금 api 응답 날라가는거 의도하고 다음과 같다고 알고있는데요!
```
{data: {memberId:1}}
```
이렇게 응답값에 응답코드를 같이 넣어주면서 mapping url 쪽은 보기 좀더 깔끔하게 하는 것도 괜찮다는 생각이 들었어요!